### ISSUE TITLE
Reporting for duty

### ISSUE THREAD

**Issue by mbicknese** [stealjs/steal-sass#21]
After reading up on the modlet workflow we started using Steal. However our setup uses SASS so we'll need this package to work. Therefore I'm reporting for maintaining this package.

**Comment by pYr0x** [comment]
Can you give us a quick overview what you want to do and how you solving the issues related to sass.
If you want we can have a short meetup and discuss the problems.

**Comment by mbicknese** [comment]
For starters there's a couple of open issues and pull requests. One of them is mine making this project compatible with the latest version of StealJS. I'd like to handle all of them properly. (So in short: maintaining it 😉)

Beyond that we could use the tilde import syntax which is also available in webpack. I made a hacky version suited to our needs and like to implementit properly for all. While I'm at it porting node-sass its include_path option seems like a good feature as well. That does require a PR for sass.js as well tho.

In conclusion I want this package to be up to date for the years to come and help solve issues where they arise.

**Comment by chasenlehara** [comment]
@mbicknese Not sure if you saw my message on Gitter, but I added you as a collaborator to this project, so you should have write access after accepting the invite.

BTW, we’re super stoked that you want to get this project going. Piggybacking on @pYr0x’s offer to meet about tackling this project’s problems, let us know if there’s anything you need. 😊
